dev-server/templates/index.mustache

```diff
@@ -24,6 +24,7 @@
     <li><a href="/document/parent-frame">Annotatable iframe test</a></li>
     <li><a href="/document/sidebar-external-container">Sidebar in external container test</a></li>
     <li><a href="/document/multi-frames">Multi-frame test</a></li>
+    <li><a href="/document/configurations">Ignore configuration test</a></li>
```

Comment:
```suggestion
    <li><a href="/document/configurations"><code>ignoreOtherConfiguration</code> test</a></li>
```

I think it might be hard for a new developer to understand what "ignore configuration" means. Naming it the way I've suggested here would hopefully make it more obvious that it refers to a specific configuration option.